src/flag_gems/ops/cross_entropy_loss.py

```diff
@@ -549,14 +615,14 @@ def backward(ctx, out_grad):
         grid = lambda meta: (N, triton.cdiv(D, meta["BLOCK_D"]))
         if tgt.ndim == inp.ndim:
             celoss_probability_bwd[grid](
-                out_grad, inp, tgt, weight, inp_grad, label_smoothing, mean_num, N, C, D
+                out_grad, inp, tgt, weight, inp_grad, label_smoothing, mean_num, C, D
             )
         elif label_smoothing == 0:
-            celoss_indice_bwd[grid](
-                out_grad, inp, tgt, weight, inp_grad, ignore_index, mean_num, N, C, D
+            celoss_indices_bwd[grid](
+                out_grad, inp, tgt, weight, inp_grad, ignore_index, mean_num, C, D
             )
         else:
-            celoss_indice_smooth_bwd[grid](
+            celoss_indices_smooth_bwd[grid](
```

Comment:
We also need the gradient of weight, if it exists.

Reply:
cross entropy loss doesn't accept weight requiring gradient.